### PR TITLE
ADEN-7942 Add tz_offset to DW tracking

### PR DESCRIPTION
**Pull request by jacekbj**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/tracking/pageInfoTracker.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ define('ext.wikia.adEngine.tracking.pageInfoTracker', [
 			'pv_unique_id': win.pvUID,
 			'prop_name': name,
 			'prop_value': value,
-			'timestamp': (new Date()).getTime()
+			'timestamp': (new Date()).getTime(),
+			'tz_offset': (new Date()).getTimezoneOffset()
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/tracking/pageInfoTracker.js]: Can we reuse `new Date()`?

File: extensions/wikia/AdEngine/js/tracking/viewabilityTracker.js [hunk 2]
```diff
@@ -26,7 +26,8 @@ define('ext.wikia.adEngine.tracking.viewabilityTracker', [
 			'line_item_id': slotFirstChildData.gptLineItemId || '',
 			'creative_id': slotFirstChildData.gptCreativeId || '',
 			'rv': slotParams.rv || 1,
-			'timestamp': (new Date()).getTime()
+			'timestamp': (new Date()).getTime(),
+			'tz_offset': (new Date()).getTimezoneOffset()
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/tracking/viewabilityTracker.js]: Can we reuse `new Date()`?

File: extensions/wikia/AdEngine/js/video/player/playerTracker.js [hunk 3]
```diff
@@ -46,6 +46,7 @@ define('ext.wikia.adEngine.video.player.playerTracker', [
 				'pv_number': pageLevelParams.pv,
 				'country': bridge.geo.getCountryCode(),
 				'timestamp': (new Date()).getTime(),
+				'tz_offset': (new Date()).getTimezoneOffset(),
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/video/player/playerTracker.js]: Can we reuse `new Date()`?

**Review by fraszczakszymon — APPROVED** [review]

File: extensions/wikia/AdEngine/js/tracking/pageInfoTracker.js (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/tracking/pageInfoTracker.js]: Yeah, can we?

File: extensions/wikia/AdEngine/js/tracking/viewabilityTracker.js (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/tracking/viewabilityTracker.js]: I like what Szymon says

File: extensions/wikia/AdEngine/js/video/player/playerTracker.js (hunk 3 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/video/player/playerTracker.js]: :3

File: extensions/wikia/AdEngine/package.json [hunk 4]
```diff
@@ -26,7 +26,7 @@
     "watch": "webpack -p -w"
   },
   "dependencies": {
-    "@wikia/ad-engine": "git+https://github.com/Wikia/ad-engine.git#v20.3.0",
+    "@wikia/ad-engine": "git+https://github.com/Wikia/ad-engine.git#0c23a2f4c83240dc79f87ac2e1a2c82305f79ee3",
```
Comment (dmnsobczak) [inline extensions/wikia/AdEngine/package.json]: Update hash after new ad-engine commits

**Comment by fraszczakszymon** [comment]
Closing this PR in favour of https://github.com/Wikia/app/pull/16452 because of mixed branches over here

**Pull request closed without merging**